frontend/src/metabase/models/components/ModelDetailPage/ModelDetailPage.styled.tsx

```diff
@@ -0,0 +1,57 @@
+import styled from "@emotion/styled";
+
+import EditableText from "metabase/core/components/EditableText";
+import Radio from "metabase/core/components/Radio";
+import BaseTabPanel from "metabase/core/components/TabPanel";
+
+import { color } from "metabase/lib/colors";
+
+export const RootLayout = styled.div`
+  display: flex;
+  flex: 1;
+  flex-direction: row;
+
+  padding: 3rem 4rem;
+  min-height: 90vh;
+`;
+
+export const ModelMain = styled.div`
+  display: flex;
+  flex: 1;
+  flex-direction: column;
+
+  padding-right: 3rem;
+`;
+
+export const ModelTitle = styled(EditableText)`
+  color: ${color("text-dark")};
+  font-weight: 700;
+  font-size: 1.25rem;
+`;
+
+export const ModelFootnote = styled.p`
+  color: ${color("text-medium")};
+  margin: 4px 0 0 4px;
+`;
+
+export const ModelHeader = styled.div`
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: flex-start;
+  width: 100%;
+`;
+
+export const TabList = styled(Radio)`
+  margin-top: 1rem;
+`;
+
+TabList.defaultProps = { variant: "underlined" };
+
+export const TabPanel = styled(BaseTabPanel)`
+  display: flex;
+  flex: 1;
+  flex-direction: column;
+
+  margin-top: 1rem;
```

Comment:
I think you also wanna lose this line to keep empty sections from taking up space

![Screen Shot 2022-09-19 at 2 01 17 PM](https://user-images.githubusercontent.com/30528226/191106317-18fcfe99-582a-4132-bfdf-9aad1ad440c3.png)
```suggestion
```